net/SslSocket.hpp

```diff
@@ -19,7 +20,7 @@
 class SslStreamSocket final : public StreamSocket
 {
 public:
-    SslStreamSocket(const std::string hostname, const int fd, bool isClient,
+    SslStreamSocket(std::string hostname, const int fd, bool isClient,
```

Comment:
de-const-ipating hostname ?

Reply:
Haha. Actually, the argument is moved, so the const doesn't help here. Clang-tidy fix.